### ISSUE TITLE
Push notifications not working even though app says they are supported.

### ISSUE THREAD

**Issue by gerg5c42g542g2c54g52c** [ChatSecure/ChatSecure-iOS#735]
Background fetch is turned on, notifications are turned on.
The push notifications settings pane has all the green ticks.
Newest ChatSecure version.
Server: creep.im (ejabberd 16.09)

**Comment by chrisballinger** [comment]
Working for me on my personal server running Prosody. I'll make an account on creep and see if I can reproduce.

**Comment by iNPUTmice** [comment]
FWIW it's fair to assume that creep.im doesn't have a working push module as there is no publicly available push module for ejabberd.

**Comment by chrisballinger** [comment]
@iNPUTmice What about [`mod_push`](https://github.com/royneary/mod_push)? Looks like its dead though.

**Comment by iNPUTmice** [comment]
@chrisballinger I don't think this was ever in a working state. Always needed a patched ejabberd. Changes were never upstreamed.
We will upstream the one running on conversations.im eventually. But there is still a lot of code cleanup to do.

**Comment by mwlczk** [comment]
Looks like @weiss is taking some actions about it:
https://github.com/processone/ejabberd/issues/1379

@gerg5c42g542g2c54g52c did you set up the creep.im server yourself?
I need some advice to set up push on my own ejabberd-instance.

**Comment by gerg5c42g542g2c54g52c** [comment]
@mwlczk No, I didn't. You can contact the admin via XMPP/E-mail at a@creep.im
